api/controller/psmdb_cluster.proto

```diff
@@ -61,3 +61,17 @@ message PSMDBClusterParams {
     }
   ];
 }
+
+// PSMDBCredentials Represents the credentials needed to connect to the cluster.
+message PSMDBCredentials {
+  // MongoDB username.
+  string username = 1;
+  // MongoDB password.
+  string password = 2;
+  // MongoDB host.
+  string host = 3;
```

Comment:
Maybe we should use our "normal" order host, port, username, password?
Do we need both host and port, or it can be a single "address" field?

Reply:
Maybe it can be just address. Can we create a ticket for that?

Reply:
The operator returns only a field named "host" and the port is (currently) mapped to the default port but I don't know if in the future there will be an option to map it to a different port.